clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/caching/Keys.java

```diff
@@ -252,6 +248,11 @@ public ApplicationCacheKey(String[] parts) {
       name = parts[3];
     }
 
+    @Override
+    public LogicalKind getLogicalKind() {
```

Comment:
Just curious, do we have to add a getter here because logicalKinds is static/final, or is there another reason the lombok getter isn't doing the trick?

Reply:
Good question---I couldn't get Lombok to do it because the getter it was creating was static, and you can't override an instance method with a static method.  So I had to write my own instance method even though it only references a static variable.

Reply:
:+1: thank you for explaining!